### PR TITLE
feat: add Parquet tap from AE Nv maintainer

### PR DESCRIPTION
**Pull request by ThibauldC**
Forked and updated tap-parquet from dataops-tk variant

### CODE REVIEW

File: _data/meltano/extractors/tap-parquet/ae-nv.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- file
+- meltano_sdk
+label: Parquet
+logo_url: /assets/logos/extractors/parquet.png
+maintenance_status: active
+name: tap-parquet
+namespace: tap_parquet
+next_steps: ''
+pip_url: git+https://github.com/AE-nv/tap-parquet.git
+quality: unknown
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-parquet/ae-nv.yml:22]:
```suggestion
quality: silver
```

**Review by pnadolny13 — COMMENTED** [review]
@ThibauldC this is great! Thanks for cleaning this tap up. Since its the better version of the 2 and it doesnt look like the dataops-tk variant will be maintained, can you set yours to the default in https://github.com/meltano/hub/blob/0ba3e0690b0e60c2bfc5dcc4cfc6b48cc96fa7e2/_data/default_variants.yml#L352?

File: _data/meltano/extractors/tap-parquet/ae-nv.yml (hunk 1 above)
Comment (ThibauldC) [reply _data/meltano/extractors/tap-parquet/ae-nv.yml]: Done 😄

**Review by pnadolny13 — APPROVED** [review]
🚀

**Pull request merged**